qualtran/bloqs/mcmt/controlled_via_and.py

```diff
@@ -126,6 +126,29 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> set['BloqCountT']:
 
         return set(counts.items())
 
+    def get_ctrl_system(self, ctrl_spec: 'CtrlSpec') -> tuple['Bloq', 'AddControlledT']:
+        """flatten nested controlled bloqs"""
```

Comment:
Either write a complete docstring or remove or relegate to a line comment

Reply:
removed